api.go

```diff
@@ -53,9 +55,17 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	case "/requests":
 		if r.Method != "POST" {
 			http.Error(w, "everdeen: Method Not Allowed", http.StatusMethodNotAllowed)
+			return
 		}
 
 		s.findRequests(w, r)
+	case "/reset_all":
```

Comment:
Going to stick with DELETE and make the endpoint look more REST-ful because ruby net/http doesn't natively support PURGE.